scripts/patch-realsense-ubuntu-L4T.sh

```diff
@@ -11,6 +11,37 @@ exec 2> /dev/null
 con_dev=$(ls /dev/video* | wc -l)
 exec 2>&3
 
+function DisplayNvidiaLicense {
+    patches_revison=$1
+
+    # verify that curl is installed
+    if  ! which curl > /dev/null  ; then
+      echo "curl is not installed."
+      echo "curl can be installed by 'sudo apt-get install curl'."
+      exit 1
+    fi
+
+    # By default referencing license agreement of JP 5.0.2
+    license_path="https://developer.download.nvidia.com/embedded/L4T/r35_Release_v1.0/Release/Tegra_Software_License_Agreement-Tegra-Linux.txt"
+
+    if [ "5.1.2" = ${patches_revison} ]; then
```

Comment:
@Nir-Az, shall I remove "5.1.2" as it is not supported currently. or is it planned to support?

Reply:
Let's keep it